components/prism-dot.js

```diff
@@ -0,0 +1,77 @@
+// https://www.graphviz.org/doc/info/lang.html
+
+(function (Prism) {
+
+	var ID = '(?:' + [
+		// an identifier
+		/\b[a-zA-Z_][\w\x80-\x9F\xA1-\xFF]*/.source,
+		// a number
+		/-?(?:\.\d+|\b\d+(?:\.\d*)?)/.source,
+		// a double-quoted string
+		/"(?:[^\\"]|\\[\s\S])*"/.source,
```

Comment:
@RunDevelopment , what do you think about the following pattern?

```
"[^"\\]*(?:\\"?[^"\\]*)*"
```

I think, it’s a bit faster, but I’m not sure in correctness. Well, at least `pattern-tests.js` doesn’t complain :)

Reply:
You're right! I tested it and it's about 4x faster when no/few escapes are present (the common case) and 2x slower when the string contains only escapes. It's a lot faster in the common case, so that's a definite yes from me!

The correct version is `/"[^"\\]*(?:\\[\s\S][^"\\]*)*"/` btw.